python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -1555,4 +1599,157 @@ def sqrt(x, out=None, **kwargs):
     return _unary_func_helper(x, _npi.sqrt, _np.sqrt, out=out, **kwargs)
 
 
+@set_module('mxnet.symbol.numpy')
+def reciprocal(x, out=None, **kwargs):
```

Comment:
Please follow this example to add description. The input is either a `_Symbol` or a scalar value, instead of `ndarray`.
https://github.com/apache/incubator-mxnet/pull/15377/files#diff-e9b526dba36aa6f2e7dbb6c4d49e9822R1584